packages/sdk/src/modules/did/index.ts

```diff
@@ -12,20 +12,30 @@ import { CreateStorageKeysEnum } from '../../types';
 import { Base } from '../base';
 
 export interface CustomDocumentFields {
-  services: DocumentService[];
+  verifications?: Verification[],
+  signatures?: Signature[],
+  services?: Service[];
 }
 
-type DocumentService = {
+type Verification = {
```

Comment:
In our backend what do we populate for the verification method? Is the user suppose to do this themselves? Currently we have it so the id is auto generate, controller is set to the address calling the function, and public key multibase is set.